app/Http/Requests/UserFormRequest.php

```diff
@@ -27,6 +27,7 @@ public function rules()
             'name' => 'required|min:3|max:255',
             'email' => 'required|unique:users,email,' . $this->id,
             'password' => 'required|string|min:6',
+            'password_confirm' => 'required|string|min:6|same:password'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/UserFormRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma